src/main/java/com/happeningnow/controller/OrganizerController.java

```diff
@@ -0,0 +1,51 @@
+package com.happeningnow.controller;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.service.ServiceOrganizer;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/organizer")
+public class OrganizerController {
+
+    private final ServiceOrganizer serviceOrganizer;
+
+    public OrganizerController(ServiceOrganizer serviceOrganizer){
+        this.serviceOrganizer = serviceOrganizer;
+    }
+
+    @PostMapping("/save")
+    public ResponseEntity<Organizer> save(@RequestBody Organizer organizer){
+        Organizer saveOrganizer = serviceOrganizer.save(organizer);
+        return new ResponseEntity<>(saveOrganizer,HttpStatus.CREATED);
+    }
+
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public ResponseEntity<Optional<Organizer>> findById(@PathVariable UUID id) {
+        Optional<Organizer> organizer = serviceOrganizer.findById(id);
```

Comment:
we don't usually return an Optional. 

I want you to throw an exception in case the ID is not found in the database